packages/integrations/node/README.md

```diff
@@ -150,6 +150,10 @@ You can pass the path to your key and certification via the environment variable
 SERVER_KEY_PATH=./private/key.pem SERVER_CERT_PATH=./private/cert.pem node ./dist/server/entry.mjs
 ```
 
+#### Runtime environment variables
+
+If an `.env` file is present when the build process is run, the values of environment variables provided by this file will be hard-coded in the output, same as when generating a static website. Special handling is required for values that have to be provided at run-time. Such variables must exist with any value during the build process: `RUNTIME_VALUE=placeholder astro build`. The placeholder value will be ignored by the build process, however its existence will signal to Astro that the actual value will be available as an environment variable when the built application will be run. In the case of multiple variables, you can store them in a file like `.env.runtime` and then start the build with the following command: `export $(cat .env.runtime) && astro build`. Note that these variables **must be absent** in the `.env` file.
```

Comment:
``````suggestion
If an `.env` file containing environment variables is present when the build process is run, these values will be hard-coded in the output, just as when generating a static website. 

For a value to be provided at run-time, use the following variable with any placeholder value in your `.env` file:  `RUNTIME_VALUE=placeholder astro build`.  This variable will signal to Astro that the actual value will be available when the built application is run. The placeholder value will be ignored by the build process, and Astro will use the value provided at run-time. 

In the case of multiple run-time variables, store them in a seperate file (e.g. `.env.runtime`) outside of `.env`. Start the build with the following command: 
```sh
export $(cat .env.runtime) && astro build
```
``````
I've edited this to split up the content into easier to read chunks, and tried to tighten up some wording while I'm at it. Of course, when I do this, I might often cut something important or miss a nuance, so please have a read over this as a suggested model, and don't hesitate to correct so that it is actually factually accurate!  :smile:

Reply:
Thank you, Sarah, I’m always happy to get an input from a tech writer and a native speaker!

This topic is unfortunately overly complex, and I see that I could not explain it clearly enough. For one, it’s not the single `RUNTIME_VALUE` which needs to be provided: during the build the developer needs to provide *every* value which will also come in during the execution. Another important part which lost its emphasis is this: "Note that these variables **must be absent** in the `.env` file." 

What do you think about this improved version?

>If an `.env` file containing environment variables is present when the build process is run, these values will be hard-coded in the output, just as when generating a static website. 

>Special handling is required for values that have to be provided at run-time. During the build, they **must be absent** from the `.env` file. But you also need to tell Astro which environment variables to expect at run-time. For that you need to provide them during the build process: `VALUE1=placeholder VALUE2=placeholder astro build`. Feel free to replace `placeholder` with any non-falsy value, it will be ignored anyway.

>Providing multiple values in this way could be cumbersome. For convenience, store them in a seperate file (e.g. `.env.runtime`). Start the build with the following command: 
```sh
export $(cat .env.runtime) && astro build
```

Reply:
Sorry for the long wait. Our docs reviewing/editing has had a huge backlog and I'm picking this up for Astro Docs Maintainers!

This is looking much better! I have a few suggestions based on your revision, of which I've thrown in a new review.